tests/run-pass/current_dir_with_isolation.stderr

```diff
@@ -0,0 +1,12 @@
+warning: `getcwd` was made to return an error due to isolation
```

Comment:
This fails on Windows because the output is different there.
Can you add `// normalize-stderr-test` comments to the test file to make sure the output is the same across platforms? It should look something like this
```
// normalize-stderr-test "(getcwd|GetCurrentDirectoryW)" -> "$$FUN"
```